docs/content/users/quickstart.md

```diff
@@ -769,6 +750,29 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
     The [`server.js`](https://github.com/ddev/test-nodejs/blob/main/server.js) used here is a trivial Express-based Node.js webserver. Yours will be more extensive.
 
+## OpenMage
+
+=== "Git Clone"
+
+    ```bash
+    mkdir my-openmage-site && cd my-openmage-site
+    git clone https://github.com/OpenMage/magento-lts .
```

Comment:
This does no harm, but wouldn't it be simpler to do this? Wouldn't people understand it ok?

```suggestion
    git clone https://github.com/OpenMage/magento-lts my-openmage-site
```